tests/negative/best_practices.cpp

```diff
@@ -86,8 +86,9 @@ TEST_F(VkBestPracticesLayerTest, ValidateReturnCodes) {
     format_count -= 1;
     formats.resize(format_count);
 
-    m_errorMonitor->SetDesiredFailureMsg(kInformationBit, "UNASSIGNED-BestPractices-NonSuccess-Result");
+    m_errorMonitor->SetDesiredFailureMsg(kVerboseBit, "UNASSIGNED-BestPractices-Verbose-Success-Logging");
     result = vk::GetPhysicalDeviceSurfaceFormatsKHR(gpu(), m_surface, &format_count, formats.data());
+    assert(result > VK_SUCCESS);
```

Comment:
```suggestion
    ASSERT_TRUE(result > VK_SUCCESS);
```
(it it makes sense?)

Reply:
It makes sense.